gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -240,6 +240,7 @@ MM_VerboseHandlerOutput::handleInitialized(J9HookInterface** hook, uintptr_t eve
 	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
 	MM_VerboseWriterChain* writer = _manager->getWriterChain();
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+	MM_GCExtensionsBase* ext = MM_GCExtensionsBase::getExtensions(env->getOmrVM());
```

Comment:
Do you need a new variable?  Isn't _extensions a GCExtensions?

Reply:
fixed